core/io/file_access_zip.cpp

```diff
@@ -157,12 +156,9 @@ bool ZipArchive::try_open_pack(const String &p_path, bool p_replace_files, size_
 	}
 
 	zlib_filefunc_def io;
+	zeromem(&io, sizeof(io));
```

Comment:
We removed Godot's custom `zeromem` recently so this needs to be rebased to use `memset(&io, 0, sizeof(io));`.